Files/ViewModels/SettingsViewModel.cs

```diff
@@ -28,17 +29,31 @@ public class SettingsViewModel : ObservableObject
     {
         private readonly ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
 
-        public DrivesManager DrivesManager { get; }
+        public DrivesManager DrivesManager
```

Comment:
I agree @yaichenbaum we should probably remove the drives from settings, they don't have much business here

Reply:
Maybe a challenge for another day :)